tests/test_microservice/test_cli/test_cli.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+
+from typer.testing import (
+    CliRunner,
+)
+
+from minos.microservice.cli import (
+    app,
+)
+from tests.utils import (
+    BASE_PATH,
+)
+
+runner = CliRunner()
+
+
+class TestCli(unittest.TestCase):
```

Comment:
I didn't know that testing `Typer` CLIs can be done so clean, this approach look really good @vladyslav-fenchak! 🚀